code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -109,10 +110,28 @@ var/bomb_set
 				if(welder.remove_fuel(1,user))
 					if(do_after(user,50,target=src))
 						playsound(loc, 'sound/items/Deconstruct.ogg', 100, 1)
-						user << "<span class='notice'>You cut into [src]'s warhead. You can see the core's green glow.</span>"
-						deconstruction_state = NUKESTATE_CORE_EXPOSED
+						user << "<span class='notice'>You cut into [src]'s warhead. The last containment plate seems to be stuck...</span>"
+						deconstruction_state = NUKESTATE_BREACHED
 						update_icon()
-						SSobj.processing += core
+				return
+		if(NUKESTATE_BREACHED)
+			if(istype(I, /obj/item/weapon/wrench))
+				playsound(loc, 'sound/effects/bang.ogg', 100, 1)
+				user << "<span class='notice'>You hit the containment plate with [I].</span>"

```

Comment:
With this message some players will assume that it's the standard hit message and that the wrench doesn't do anything. A verb other than "hit" would be nice.